src/defaults/render-actions.js

```diff
@@ -20,33 +18,39 @@ const ActionsWrapper = styled.div`
   padding-bottom: 80px;
 `
 
-const Actions = ({
-  isVideoInputSupported,
-  isInlineRecordingSupported,
-  thereWasAnError,
-  isRecording,
-  isCameraOn,
-  streamIsReady,
-  isConnecting,
-  isRunningCountdown,
-  isReplayingVideo,
-  countdownTime,
-  timeLimit,
-  showReplayControls,
-  replayVideoAutoplayAndLoopOff,
-  useVideoInput,
-
-  onTurnOnCamera,
-  onTurnOffCamera,
-  onOpenVideoInput,
-  onStartRecording,
-  onStopRecording,
-  onPauseRecording,
-  onResumeRecording,
-  onStopReplaying,
-  onConfirm
-}) => {
-  const renderContent = () => {
+class Actions extends PureComponent {
```

Comment:
Made this a class component so that we can at least work with state within it (and making it a functional component threw some errors since the parent was a class component 🤷 )

Reply:
Great note!